cli/src/print.rs

```diff
@@ -47,6 +47,13 @@ impl<W: Write> Print<W> {
 
                 writeln!(self.output, "{}\n", table)?;
             }
+            Payload::ShowIndexes(indexes) => {
+                let mut table = get_table(vec!["Index Name", "Order"]);
+                for index in indexes {
+                    table.add_row([index.0, index.1.to_string()]);
+                }
+                writeln!(self.output, "{}\n", table)?;
+            }
```

Comment:
Could you add a test case for `SHOW INDEXES --` CLI result?

Reply:
+1.. done!